packages/ui/src/components/git-commit-dialog/git-commit-dialog.tsx

```diff
@@ -133,25 +136,26 @@ export const GitCommitDialog: FC<GitCommitDialogProps> = ({
           <Dialog.Title>Commit Changes</Dialog.Title>
         </Dialog.Header>
 
-        <form className="flex flex-col gap-y-7 pb-4" onSubmit={handleSubmit(onSubmit)}>
+        <FormWrapper {...formMethods} onSubmit={handleSubmit(onSubmit)}>
           {isFileNameRequired && (
-            <Input
+            <FormInput.Text
               id="fileName"
               label="File Name"
               {...register('fileName')}
               placeholder="Add a file name"
-              size="md"
-              error={errors.fileName?.message?.toString()}
+              theme={errors.fileName?.message ? 'danger' : 'default'}
+              error={errors.fileName?.message}
               autoFocus
             />
           )}
-          <Input
+          <FormInput.Text
+            autoFocus={!isFileNameRequired}
             id="message"
             label="Commit Message"
             {...register('message')}
             placeholder={commitTitlePlaceHolder ?? 'Add a commit message'}
-            size="md"
-            error={errors.message?.message?.toString()}
+            theme={errors.message?.message ? 'danger' : 'default'}
+            error={errors.message?.message}
```

Comment:
Doesn't `FormInput.Text` connect to the form context errors automatically?

Reply:
Yes, removed it.